src/object_properties.cpp

```diff
@@ -109,7 +109,7 @@ void ObjectProperties::serialize(std::ostream &os) const
 	writeF1000(os, automatic_face_movement_max_rotation_per_sec);
 	os << serializeString(infotext);
 	os << serializeString(wield_item);
-	writeU8(os, can_zoom);
+	writeF1000(os, zoom_fov);
```

Comment:
Is it ok to just change this and not move it? The comment for this section says to only add stuff to the bottom and not remove anything, but maybe that doesn't apply since we are breaking compatibility and 'can zoom' was added during 0.5.0dev.

Reply:
This will cause an offset in the ObjectProperties' network packet (before: 8 bits, now: 32 bits)
Yes, we are in backwards compat breakage but nevertheless you should either raise the version number here from 2 to 3 or deprecate this single byte.

Reply:
Raising version just for this seems excessive.
Maybe it's best to keep using 'can zoom' also.
By deprecation i assume you mean keep writing 'can zoom' but set it to a fixed bool and don't read it? I don't like the messiness of that.